File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    let pc = 0;
    let yc = 0;
    
    for(let i of s.toLowerCase()) {
        if(i === 'p') {
            pc += 1;
        }else if (i === 'y') {
            yc += 1;
        }
    }

    return pc === yc;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int pc = 0;
        int yc = 0;
        for(String i: s.toLowerCase().split("")) {
            if(i.equals("p")) {
                pc++;
            }else if(i.equals("y")) {
                yc++;
            }
        }

        return pc == yc;
    }
}